src/cmds/scripts/pbs_preuninstall

```diff
@@ -61,6 +61,12 @@ server|execution)
         echo "have systemd"
         systemctl disable pbs
         rm -f /usr/lib/systemd/system-preset/95-pbs.preset
+        systemctl stop pbs_jobs.service
```

Comment:
If pbs_jobs.service does not exist, would this call error out?

Reply:
It will throw an error message but the preuninstall script will continue to run to the end. I think your concern is the same as what Vincent raise below: the service name can change with cgroup-prefix. I am working on this.